doc/styles/Vocab/ANSYS/accept.txt

```diff
@@ -1,9 +1,7 @@
-ANSYS
-Ansys
-ansys
 Granta MI
 PyGranta
 RecordLists
 API
 Makefile
 jupytext
+(?i)ansys
```

Comment:
Doesn't that make "aNsys", "ansYS", etc valid ?

Reply:
It does, but according to the vale documentation, having multiple versions of the same string with different casing isn't supported. So we'll just need to keep an eye out for those scenarios.

Reply:
If Ansys wasn't sometimes used in all caps, it wouldn't be an issue. But in our case, yes, it's not ideal.